File: java_apx/src/main/java/net/sympower/citizen/apx/config/ProviderEndpointsProperty.java
package net.sympower.citizen.apx.config;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Getter
@Setter
@Component
@ConfigurationProperties("provider-endpoints")
public class ProviderEndpointsProperty {
    private String someProviderGet;
    private String someProviderPost;
}

File: kotklin_apx/src/main/kotlin/net/sympower/citizen/apx/config/ProviderEndpointsProperty.kt
package net.sympower.citizen.apx.config

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component
@ConfigurationProperties("provider-endpoints")
class ProviderEndpointsProperty {
    var someProviderPost: String = ""
    var someProviderGet: String = ""
}